src/main/resources/jenkins/plugins/git/GitStep/help.html

```diff
@@ -0,0 +1,11 @@
+<div>
```

Comment:
Yeah, probably not necessary, but I like to have a root element in the file. Should be harmless anyway.

Note that this file is unmodified, it is just moved from another repository.